app/helpers/accessible_navigation_helper.rb

```diff
@@ -1,5 +1,5 @@
 module AccessibleNavigationHelper
-  def link_to_accessible(name = nil, options = nil, html_options = {}, &)
+  def govuk_link_to(name = nil, options = nil, html_options = {}, &)
```

Comment:
Can this not just be deleted now?

If we rename it, won't it clash with the 'official' helper?